.github/workflows/publish.yaml

```diff
@@ -0,0 +1,20 @@
+name: Publish to NPM
+on:
+  push:
+    branches:
+      - release
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          registry-url: https://registry.npmjs.org/
+      - run: node --version
+      - run: npm install
```

Comment:
I would expect `npm ci` to be better here, as you've used in your additions to `.github/workflows/ci.yaml`. If you agree, could you also change all the existing  `npm install` commands there to `npm ci`? 

I notice that [the GitHub](https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#publishing-packages-to-the-npm-registry) docs use `npm install` here, so there may be a reason behind this.

Reply:
You're right, I think `npm ci` is the move for anything running on build servers. Based on [the official doc](https://docs.npmjs.com/cli/v8/commands/npm-ci), I think this is the main difference/benefit for us:

> If dependencies in the package lock do not match those in package.json, npm ci will exit with an error, instead of updating the package lock.

I've replaced `npm install` with `npm ci` in both YAML files: https://github.com/nextstrain/auspice/pull/1434/commits/cc8fa32bf059340946bcfc769600a9d9a6f662be